Documentation/ceph-quickstart.md

```diff
@@ -25,6 +25,22 @@ In order to configure the Ceph storage cluster, at least one of these local stor
 - Raw partitions (no formatted filesystem)
 - PVs available from a storage class in `block` mode
 
+You can confirm whether your partitions or devices are formatted filesystems with the following command.
+
+```
+# lsblk -f
+NAME                  FSTYPE      LABEL UUID                                   MOUNTPOINT
+vda
+└─vda1                LVM2_member       eSO50t-GkUV-YKTH-WsGq-hNJY-eKNf-3i07IB
+  ├─ubuntu--vg-root   ext4              c2366f76-6e21-4f10-a8f3-6776212e2fe4   /
+  └─ubuntu--vg-swap_1 swap              9492a3dc-ad75-47cd-9596-678e8cf17ff9   [SWAP]
+vdb
+```
+
+If the `FSTYPE` field is not empty, there is a filesystem on top of the corresponding device. In this case, you can use vdb for Ceph and can't use vda and its partitions.
+
+You can delete your filesystem with `dd if=/dev/zero of=<disk name> bs=1M count=100 oflag=direct,dsync`. Of course, you must check there is no important data in this filesystem beforehand.
```

Comment:
I would not recommend that, they can always look it up on Google if they want to?

Reply:
Ah, indeed.

Reply:
So, I want to confirm what you said correctly. You mean the 1st paragraph is enough and the 2nd paragraph is not necessary?

Reply:
Correct. Just remove `You can delete your filesystem with `dd if=/dev/zero of=<disk name> bs=1M count=100 oflag=direct,dsync`. Of course, you must check there is no important data in this filesystem beforehand.`